sdk/keyvault/Azure.Security.KeyVault.Administration/tests/AccessControlClientLiveTests.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Administration.Models;
+using static Azure.Security.KeyVault.Administration.KeyVaultAdministrationConstants;
+using NUnit.Framework;
+using System.Linq;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class AccessControlClientLiveTests : AccessControlTestBase
+    {
+        private const string pricipalID = "4ae6842b-2a3e-4919-8305-7db89f3d6edd";
+        private const string roleName = "Azure Key Vault Managed HSM Administrator";
+        private const string roleAssignmentId = "e7ae2aff-eb17-4c9d-84f0-d12f7f468f16";
+
+        public AccessControlClientLiveTests(bool isAsync) : base(isAsync, RecordedTestMode.Playback /* To record tests, add this argument, RecordedTestMode.Record */)
```

Comment:
`RecordedTestMode.Playback` is intentionally hard coded because we don't yet have the ability to stand up new instances of MHSM.